en/simulation/failsafes.md

```diff
@@ -1,26 +1,49 @@
+
+> **Note** The parameter gets reset if SITL is restarted. 
+  To prevent that from happening remove the line `param set NAV_DLL_ACT 0` from the [SITL startup script](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d-posix/rcS).
 
 ## RC Link Loss
-By default unavailability of any remote remote control (RC) data is ignored to make the simulation conveniently usable without additonal dependencies.
 
-To test RC loss set the parameter `NAV_RCL_ACT` to the desired reaction according to the possibilities listed in the [parameter reference](advanced/parameter_reference.html) e.g. to the value `2` to make the vehicle return to launch.
+The *RC Link Loss* failsafe (unavailability of data from a remote control) is ignored by default.
+This makes the simulation usable without additional hardware.
+
+Set the parameter [NAV_RCL_ACT](../advanced/parameter_reference.md#NAV_RCL_ACT) to the desired failsafe action to enable the failsafe. 
+For example, set to `2` to make the vehicle return to launch.
+
+> **Note** The parameter gets reset if SITL is restarted. 
+  To prevent that from happening remove the line `param set NAV_RCL_ACT 0` from the [SITL startup script](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d-posix/rcS).
 
-> **Note** This parameter gets reset once SITL is restarted. To prevent that from happening remove the line `param set NAV_RCL_ACT 0` from the [SITL startup script](https://github.com/PX4/Firmware/blob/1d86b7570a75844375cb4c7d75731ec33826f173/ROMFS/px4fmu_common/init.d-posix/rcS).
 
 ## Low Battery
-By default the simulated battery only depletes to 50% of its capacity to enable testing of any battery indication but not triggering any low battery reactions that might interrupt ongoing simulation testing.
-To change this minimal percentage value change [this line](https://github.com/PX4/Firmware/blob/9d67bbc328553bbd0891ffb8e73b8112bca33fcc/src/modules/simulator/simulator_mavlink.cpp#L330).
 
-To control how fast the battery depletes to this value use the parameter `SIM_BAT_DRAIN`. By changing this configuration in flight you can also test regaining capacity to simulate inaccurate battery state estimation or in air charging technology.
+The simulated battery is implemented to never run out of energy, and by default only depletes to 50% of its capacity. 
+This enables testing of battery indication in GCS UIs without triggering low battery reactions that might interrupt other testing.
+
+To change this minimal battery percentage value change [this line](https://github.com/PX4/Firmware/blob/9d67bbc328553bbd0891ffb8e73b8112bca33fcc/src/modules/simulator/simulator_mavlink.cpp#L330).
+
+To control how fast the battery depletes to the minimal value use the parameter [SIM_BAT_DRAIN](../advanced/parameter_reference.md#SIM_BAT_DRAIN).
+
+> **Tip** By changing this configuration in flight, you can also test regaining capacity to simulate inaccurate battery state estimation or in-air charging technology.

```

Comment:
@MaEtUgR How can you test this in flight? Changing the percentage is a code change, and SIM_BAT_DRAIN does not appear to have a negative value (ie for charging)

Reply:
`SIM_BAT_DRAIN` is the time in seconds until the battery empties to the value you specified. If it emptied let's say in 60s and you increase the value to 1000s the battery voltage will rise again 🔋
I should also make a simulation only parameter out of the other setting sucht that recompilation is not needed.